Pipfile

```diff
@@ -0,0 +1,60 @@
+[[source]]
```

Comment:
I don't get what happened here - you have a `requirements.txt`, but also a `Pipfile` and a `Pipfile.lock` - but the `Pipfile` has all the packages pinned to the dot, probably not what you want with it.

And then there's also the `setup.py` - what do you need this for? We won't publish or build this as a package/egg anywhere yet, and I don't want us to write code we're not using immediately, because it will end up with bugs we don't know about.

Pick one single dependency management system and remove the others

Reply:
Yeah that was messy as hell. I removed the `requirements.txt` for good. I also upgraded everything.
I added `setup.py` because the pytest docs recommend it. I got a little lost with the import structure for the tests. But I removed it and the tests are working.